drone-localization/utils.py

```
@@ -0,0 +1,63 @@
+    last_model_name = os.path.basename(get_model_list(dirname, 'net'))
+    if last_model_name is None:
+        raise FileNotFoundError(f"No .pth files found in {dirname}")
+    
+    epoch = last_model_name.split('_')[1].split('.')[0]
+    if epoch != 'last':
+        epoch = int(epoch)
+    
+    # Загружаем конфигурацию
+    config_path = os.path.join(dirname, 'opts.yaml')
+    with open(config_path, 'r') as stream:
+        config = yaml.load(stream, Loader=yaml.FullLoader)
```

Ruff (0.15.12)

[error] Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)